protocol/src/main/java/io/camunda/zeebe/protocol/record/value/BpmnElementType.java

```diff
@@ -51,9 +51,6 @@ public enum BpmnElementType {
   MULTI_INSTANCE_BODY(null),
   CALL_ACTIVITY("callActivity"),
 
-  // TODO (saig0): remove element type for testing - #6202
-  TESTING_ONLY(null),
-
```

Comment:
:wastebasket: Finally gone :sweat_smile: 

:x: Currently, the [build is failing](https://ci.zeebe.camunda.cloud/blue/organizations/jenkins/camunda%2Fzeebe/detail/pi_anywhere_unsupported_elements/2/pipeline#step-82-log-861) because of this change. We need to update `protocol/revapi.json` because it is detected as an API change.